app/views/enterprises/shop.html.haml

```diff
@@ -6,7 +6,8 @@
   = current_distributor.logo.url
 
 - content_for :injection_data do
-  = inject_enterprise_shopfront(@enterprise)
+  - cache "enterprises/shop/inject_enterprise_shopfront-#{@enterprise.id}", expires_in: CacheService::SHOPS_EXPIRY do
```

Comment:
same here, if SHOPS_EXPIRY is in the CacheService, at least the key should be in the service as well.